core/runtimes/FiberRuntime/index.js

```diff
@@ -139,28 +139,17 @@ FiberRuntime.prototype._hasDeadLock = function () {
 FiberRuntime.prototype._genInitialQueue = function () {
   var self = this;
   var queue = [];
-  var keys = Object.keys(self._list);
```

Comment:
It's a function that you pass to the [`reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) method on a list.

In general, it reduces a list to a single value.

Basically, you take a list, take the first element and some initial value and perform a function on them. You then take the second element and apply the function again to the third element and the result of the first application and so on and so on.

A simple example:

```
[1,2,3,4].reduce(function(sum, number) {
  return sum + number;
}, 0);
```
This performs `(0 + 1) + 2) +3) + 4)` to give you `10`.

Actually, there's an even better option for reducers assessing a boolean value for a list and thats `every` (or `some`). This returns `true/false` based on a Predicate like `(!inport[1].conn instanceof IIPConnection)`.

Fun!